app/Http/Controllers/HomeController.php

```diff
@@ -25,4 +25,37 @@ public function tours ()
 
     	return view('book_tour.pages.tours', compact('tours'));
     }
+
+    public function redirect_route ($route = null, $message = null)
```

Comment:
```suggestion
    public function redirectRoute ($route = null, $message = null)
```
sửa hết tên function